### PR TITLE
#q7u6bES5 Working on article CRUD

### PR DESCRIPTION
**Pull request by tuyizerejean**
## What does this PR do?

-  This PR contains CRUD of article

### clone this repo
In the terminal run "npm run dev" 

- #### In postman refer to this api | link "http://localhost:3000/api/v1/articles" and use post ,get and DELETE , PUT followed by ID
####  In body Click the form-data section and pass in this key and value pair

-  #### title : "Add a title here"

- #### content: "add the message here"

- #### Image : "Upload the image "

- #### Change the text to file on uplloading the image

## Any background context you want to provide?

- No

## What are the relevant pivotal tracker stories?

- https://trello.com/c/q7u6bES5/22-blog

Screenshots (if appropriate)

d0bc0da7c94
RetrieveID
4.png)
update

### CODE REVIEW

File: learn-express/routes.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+router.delete("/posts/:id", async (req, res) => {
+  try {
+    await Post.deleteOne({ _id: req.params.id });
+    res.status(204).send();
+  } catch {
+    res.status(404);
+    res.send({ error: "Post doesn't exist!" });
+  }
+});
```
Comment (ncutixavier) [inline learn-express/routes.js]: Reamove all unnecessary code

**Review by ncutixavier — CHANGES_REQUESTED** [review]
Remove all unused code

**Comment by tuyizerejean** [comment]
I have  removed the unnecessary code i think it was the error status

File: learn-express/routes.js (hunk 1 above)
Comment (tuyizerejean) [reply learn-express/routes.js]: I have  removed the unnecessary code i think it was the error status

Comment (ncutixavier) [reply learn-express/routes.js]: Remove learn-expresss....

Comment (tuyizerejean) [reply learn-express/routes.js]: yeah i was confused with that word of removing unwanted code but i have removed it

**Review by ncutixavier — APPROVED** [review]

**Pull request merged**